python/lib/dependabot/python/file_parser.rb

```diff
@@ -34,6 +37,9 @@ class FileParser < Dependabot::FileParsers::Base
         InvalidRequirement ValueError RecursionError
       ).freeze
 
+      DEFAULT_PACKAGE_MANAGER = "pip"
```

Comment:
Can you read this one from directly package manager `NAME` constant?

Reply:
its been read from name constant , this is redundant, so i removed it